Demo/UIKit/Info.plist

```diff
@@ -73,6 +73,56 @@
 	<false/>
 	<key>LSApplicationQueriesSchemes</key>
 	<array>
+		<string>twint-issuer50</string>
+		<string>twint-issuer49</string>
+		<string>twint-issuer48</string>
+		<string>twint-issuer47</string>
+		<string>twint-issuer46</string>
+		<string>twint-issuer45</string>
+		<string>twint-issuer44</string>
+		<string>twint-issuer43</string>
+		<string>twint-issuer42</string>
+		<string>twint-issuer41</string>
+		<string>twint-issuer40</string>
+		<string>twint-issuer39</string>
+		<string>twint-issuer38</string>
+		<string>twint-issuer37</string>
+		<string>twint-issuer36</string>
+		<string>twint-issuer35</string>
+		<string>twint-issuer34</string>
+		<string>twint-issuer33</string>
+		<string>twint-issuer32</string>
+		<string>twint-issuer31</string>
+		<string>twint-issuer30</string>
+		<string>twint-issuer29</string>
+		<string>twint-issuer28</string>
+		<string>twint-issuer27</string>
+		<string>twint-issuer26</string>
+		<string>twint-issuer25</string>
+		<string>twint-issuer24</string>
+		<string>twint-issuer23</string>
+		<string>twint-issuer22</string>
+		<string>twint-issuer21</string>
+		<string>twint-issuer20</string>
+		<string>twint-issuer19</string>
+		<string>twint-issuer18</string>
+		<string>twint-issuer17</string>
+		<string>twint-issuer16</string>
+		<string>twint-issuer15</string>
+		<string>twint-issuer14</string>
+		<string>twint-issuer13</string>
+		<string>twint-issuer12</string>
+		<string>twint-issuer11</string>
+		<string>twint-issuer10</string>
+		<string>twint-issuer9</string>
+		<string>twint-issuer8</string>
+		<string>twint-issuer7</string>
+		<string>twint-issuer6</string>
+		<string>twint-issuer5</string>
+		<string>twint-issuer4</string>
+		<string>twint-issuer3</string>
+		<string>twint-issuer2</string>
+		<string>twint-issuer1</string>
 		<string>weixinULAPI</string>
```

Comment:
Did you test `weixinULAPI` and the following one? They should not be handled now I think according to Apple.

Reply:
`weixinULAPI` and the following one are not related to Twint.

Reply:
`weixinULAPI` is for WeChat

Reply:
We now have 52 entries in the plist file and Apple supports 50. What would happen if we try paying via WeChat?

Reply:
From [the docs](https://developer.apple.com/documentation/uikit/uiapplication/1622952-canopenurl):
> If you link your app against an earlier version of iOS but it is running in iOS 9.0 or later, you can call this method up to 50 times. After reaching that limit, subsequent calls always return false.

Reply:
IMHO - this doesn't sound like Adyen problem 😅